pkg/resourcegroup/controllers/resourcemap/update_metrics_test.go

```diff
@@ -0,0 +1,259 @@
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcemap
+
+import (
+	"context"
+	"testing"
+
+	"github.com/GoogleContainerTools/config-sync/pkg/api/configsync"
+	"github.com/GoogleContainerTools/config-sync/pkg/api/kpt.dev/v1alpha1"
+	"github.com/GoogleContainerTools/config-sync/pkg/resourcegroup/controllers/metrics"
+	"github.com/GoogleContainerTools/config-sync/pkg/testing/testmetrics"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func TestResourceMapUpdateMetrics(t *testing.T) {
```

Comment:
Could this test be structured as a data driven test? That would help the readability

Reply:
It looks like it's still not a data driven test - is that intentional?

Reply:
EDIT: I was looking at a stale commit, my mistake